### PR TITLE
restructured config files

### PR DESCRIPTION
**Pull request by t0ffel**

### CODE REVIEW

**Comment by t0ffel** [comment]
@portante changed according to the discussion

File: ansible/roles/fluentd-node-collector/tasks/cleanup_logging.yaml [hunk 1]
```diff
@@ -2,26 +2,29 @@
 - name: "Got to logging project"
```
Comment (portante) [inline ansible/roles/fluentd-node-collector/tasks/cleanup_logging.yaml]: s/`Got to`/`Go to`/

File: ansible/roles/fluentd-node-collector/tasks/deploy_logging.yaml [hunk 2]
```diff
@@ -20,10 +20,10 @@
 - name: "Process roles template and create objects"
   shell: "oc process fluentd-roles-template | oc create -f - "
 
-- name: "Import fluentd image as imagestream"
-  shell: >
-    oc import-image logging-fluentd --from=docker.io/t0ffel/logging-fluentd
-    --all --confirm
+# - name: "Import fluentd image as imagestream"
```
Comment (portante) [inline ansible/roles/fluentd-node-collector/tasks/deploy_logging.yaml]: Why keep this commented out?

File: ansible/roles/fluentd-node-collector/templates/fluentd-daemonset.yaml [hunk 3]
```diff
@@ -123,17 +119,10 @@ items:
...
-    value: "logging-es"
-  -
-    description: "Port number for reaching fluentd normalizer to write cluster logs"
-    name: FLUENTD_NORMALIZER_PORT
-    value: "9200"
-  -
     description: "The prefix of the image to use."
     name: IMAGE_PREFIX
-    value: "atomic-registry.usersys.redhat.com:5000/t0ffel/"
+    value: "docker.io/t0ffel/"
```
Comment (portante) [inline ansible/roles/fluentd-node-collector/templates/fluentd-daemonset.yaml:124]: You might want to consider putting these choices in the README for folks to track down.

File: ansible/roles/fluentd-node-collector/vars/main.yaml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-ansible_ssh_user: root
+#ansible_ssh_user: cloud-user
```
Comment (portante) [inline ansible/roles/fluentd-node-collector/vars/main.yaml]: Why comment this out and then keep it?  Do we need it for something?

File: configs/node-collector/journal/filter-tagging.conf [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<filter **>
+  @type record_transformer
+  <record>
+    tags "ADD-SPACE-DELIMITED-TAGS-HERE"
```
Comment (portante) [inline configs/node-collector/journal/filter-tagging.conf:4]: Do we need tags specified both here and in the catchall conf file of the same name?

**Review by portante — COMMENTED** [review]
Looks good overall.

File: ansible/roles/fluentd-node-collector/tasks/cleanup_logging.yaml (hunk 1 above)
Comment (t0ffel) [reply ansible/roles/fluentd-node-collector/tasks/cleanup_logging.yaml]: done

File: ansible/roles/fluentd-node-collector/templates/fluentd-daemonset.yaml (hunk 3 above)
Comment (t0ffel) [reply ansible/roles/fluentd-node-collector/templates/fluentd-daemonset.yaml:124]: I've added a pointer to the README

File: ansible/roles/fluentd-node-collector/tasks/deploy_logging.yaml (hunk 2 above)
Comment (t0ffel) [reply ansible/roles/fluentd-node-collector/tasks/deploy_logging.yaml]: imagestreams are imho preferential in general to use, although it didn't work for me.. I think there are some weird bugs there.

File: ansible/roles/fluentd-node-collector/vars/main.yaml (hunk 4 above)
Comment (t0ffel) [reply ansible/roles/fluentd-node-collector/vars/main.yaml]: removed this, I think it's better to have this var in inventory, but that depends on a particular setup you run against.

File: configs/node-collector/journal/filter-tagging.conf (hunk 5 above)
Comment (t0ffel) [reply configs/node-collector/journal/filter-tagging.conf:4]: You generally may want different tags there.
They represent different flows. this one is for stuff labelled `journal`, the other one is generally for the stuff emitted by fluentd itself.
I don't know how to not duplicate this code in a nice way - same for the output/forward also a duplication there.

**Pull request merged**